### ISSUE TITLE
Implement URLSearchParams::getAll

### ISSUE THREAD

**Issue by frewsxcv** [servo/servo#5439]
This might be a candidate for 'E-Easy' since `URLSearchParams::get` is already done

https://url.spec.whatwg.org/#dom-urlsearchparams-getall

https://github.com/servo/servo/blob/master/components/script/dom/urlsearchparams.rs

**Comment by jdm** [comment]
https://github.com/servo/servo/blob/master/components/script/dom/webidls/URLSearchParams.webidl

**Comment by bdero** [comment]
I think this is blocked by #544

**Comment by Ms2ger** [comment]
Also, we need tests.

**Comment by dhananjay92** [comment]
I would like to work on this issue. Should I wait till #544 is solved?

**Comment by jdm** [comment]
Yes.

**Comment by frewsxcv** [comment]
#544 is done. Marking this as E-Easy again

**Comment by fmmartins** [comment]
Hey, I'd like to work on this one if no else is?

I added the `GetAll` function, and deleted the lines that expected to `FAIL` in the files:

tests/wpt/metadata/url/interfaces.html.ini
tests/wpt/metadata/url/urlsearchparams-getall.html.ini (actually I deleted this file)

And now tests are passing, but I'm not sure if this is the correct way of doing this?

**Comment by nox** [comment]
@fmmartins It is! Feel free to make a PR!

**Comment by fmmartins** [comment]
@nox Thanks, will do!
